src/nip46/utils/security.ts

```
@@ -0,0 +1,349 @@
+    }
+
+    // Check signer keypair
+    if (NIP46SecurityValidator.isPrivateKeyEmpty(bunkerInstance.signerKeypair.privateKey)) {
+      throw new NIP46SecurityError("Signer private key not properly initialized - cannot perform cryptographic operations");
+    }
+
+    // Validate both keypairs
+    NIP46SecurityValidator.validateUserKeypairForSigning(bunkerInstance.userKeypair);
+    NIP46SecurityValidator.validateSignerKeypairForBunker(bunkerInstance.signerKeypair);
+  }
+} 
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)